authBackEnd/apps/user/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+from authBackEnd.apps.user import models as account_models
+
```

Comment:
use an array and a for loop -> it's cleaner and scales better

Reply:
What do you mean? where?